### PR TITLE
Prepare googlechrome/lighthouse module to be synced

### PR DESCRIPTION
**Pull request by unmultimedio**
Relates to #78

### CODE REVIEW

**Comment by unmultimedio** [comment]
Tested, it syncs: https://github.com/bufbuild/modules/pull/116/files#diff-66b88475781fe3747cdd841d516aee1dcc60c8ab2e93f00a26c89a9a15b0451cR23-R26

File: modules/static/googlechrome/lighthouse/initref [hunk 1]
```diff
@@ -0,0 +1 @@
+14feb91c4b8e15c06244ee673818f7fe1985bf23
```
Comment (unmultimedio) [inline modules/static/googlechrome/lighthouse/initref:1]: Waiting to know chosen initref: https://github.com/bufbuild/modules/issues/78#issuecomment-1561948041

Comment (pgmitche) [reply modules/static/googlechrome/lighthouse/initref:1]: Just have a thought, disregard if it doesn't seem worth it. 

The latest major release (10) included some breaking changes. Would it be helpful to include the last two major releases to support the community that may not have migrated up?

Comment (unmultimedio) [reply modules/static/googlechrome/lighthouse/initref:1]: that's interesting, yeah they do semver releases, I guess we could tweak our releases syncer pkg to support regular semver and not only the wkt special case. I'll play a bit with it.

**Review by pkwarren — APPROVED** [review]

File: modules/static/googlechrome/lighthouse/initref (hunk 1 above)
Comment (unmultimedio) [reply modules/static/googlechrome/lighthouse/initref]: Done, I think it's cleaner now.

**Review by pgmitche — APPROVED** [review]
LGTM 😀

**Pull request merged**